spec/tree-view-spec.coffee

```diff
@@ -15,6 +15,9 @@ waitsForFileToOpen = (causeFileToOpen) ->
 describe "TreeView", ->
   [treeView, root, sampleJs, sampleTxt, workspaceElement] = []
 
+  selectEntry = (path) ->
```

Comment:
Usually when a file has `path = require 'path'` at the top, we try not to use `path` as a variable name since it conflicts, for example, `path.join` would be tricky to use inside this function if needed.

What do you thinking about calling it `pathToSelect` instead?